apps/ledger-live-desktop/tests/specs/speculos/accountSpeculos.spec.ts

```diff
@@ -0,0 +1,78 @@
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { PortfolioPage } from "../../models/PortfolioPage";
+import { AddAccountModal } from "../../models/AddAccountModal";
+import { Layout } from "../../models/Layout";
+import { AccountPage } from "../../models/AccountPage";
+import { AccountsPage } from "../../models/AccountsPage";
+import { Device, specs, startSpeculos, stopSpeculos } from "../../utils/speculos";
+
+test.use({ userdata: "skip-onboarding" });
+const currencies = ["Ethereum", "Ethereum Holesky"];
+let device: Device | null;
+
+test.describe.parallel("Accounts @smoke", () => {
+  for (const currency of currencies) {
+    let firstAccountName = "NO ACCOUNT NAME YET";
+    test(`[${currency}] Add account`, async ({ page }) => {
+      const portfolioPage = new PortfolioPage(page);
+      const addAccountModal = new AddAccountModal(page);
+      const layout = new Layout(page);
+      const accountsPage = new AccountsPage(page);
+      const accountPage = new AccountPage(page);
+      device = await startSpeculos(test.name, specs[currency.replace(/ /g, "_")]);
+
+      await test.step(`[${currency}] Open modal`, async () => {
+        await portfolioPage.openAddAccountModal();
+        expect(await addAccountModal.title.textContent()).toBe("Add accounts");
+        await expect.soft(addAccountModal.container).toHaveScreenshot(`open-modal.png`);
```

Comment:
as the idea with speculos implementation is to test without mocks, maybe we can get rid of screenshots base assertions
any take on this @abdurrahman-ledger ?